Scripts/make_scenario.py

```diff
@@ -0,0 +1,132 @@
+import json
+from math import sin, pi
+
+
+NAME = 'Sine Curve.json'
+
+
+class Scenario:
+    def __init__(self, glucoses, basals, boluses, carb_entries):
+        self.glucoses = glucoses
+        self.basals = basals
+        self.boluses = boluses
+        self.carb_entries = carb_entries
+
+    def json(self):
+        return {
+            'glucoseValues': [glucose.json() for glucose in self.glucoses],
+            'basalDoses': [basal.json() for basal in self.basals],
+            'bolusDoses': [bolus.json() for bolus in self.boluses],
+            'carbEntries': [entry.json() for entry in self.carb_entries]
+        }
+
+
+class Glucose:
+    def __init__(self, mgdl, date_offset):
+        self.mgdl = mgdl
+        self.date_offset = date_offset
+
+    def json(self):
+        return {
+            'mgdlValue': self.mgdl,
+            'dateOffset': self.date_offset
+        }
+
+
+class Basal:
+    def __init__(self, units_per_hour, date_offset, duration):
+        self.units_per_hour = units_per_hour
+        self.date_offset = date_offset
+        self.duration = duration
+
+    def json(self):
+        return {
+            'unitsPerHourValue': self.units_per_hour,
+            'dateOffset': self.date_offset,
+            'duration': self.duration
+        }
+
+
+class Bolus:
+    def __init__(self, units, date_offset, delivery_duration):
+        self.units = units
+        self.date_offset = date_offset
+        self.delivery_duration = delivery_duration
+
+    def json(self):
+        return {
+            'unitsValue': self.units,
+            'dateOffset': self.date_offset,
+            'deliveryDuration': self.delivery_duration
+        }
+
+
+class CarbEntry:
```

Comment:
[nit] You use `CarbEntry` here, but `Glucose`, `Basal`, and `Bolus` above (rather than `GlucoseValue`, `BasalDose`, and `BolusDose`).